src/App.js

```diff
@@ -45,17 +45,17 @@ function App() {
             <Route exact path="/:category">
```

Comment:
This is actually matching your slug. 
It doesn't matter the name you put here so much, since it still takes the first parameter but for the sake of understanding and clean code, please rename it `:slug`